pkg/customresourcestate/registry_factory_test.go

```diff
@@ -134,12 +134,15 @@ func Test_addPathLabels(t *testing.T) {
 		{name: "*", args: args{
 			obj: cr,
 			labels: map[string]valuePath{
-				"*1":  mustCompilePath(t, "metadata", "annotations"),
-				"bar": mustCompilePath(t, "metadata", "labels", "foo"),
+				"*1":             mustCompilePath(t, "metadata", "annotations"),
```

Comment:
Unrelated to the actual PR, but why does `*1` yield `"qux": "quxx"`. Is this the way we tell the code to take the first annotation?